includes/razorpay-webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
```

Comment:
We should do an early `die` as well, so other plugins and wordpress does not interfere with our webhooks.